tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -247,6 +248,14 @@ func (iptConfigurator *IptablesConfigurator) shortCircuitKubeInternalInterface()
 	}
 }
 
+func (iptConfigurator *IptablesConfigurator) shortCircuitExcludeInterfaces() {
+	for _, excludeInterface := range split(iptConfigurator.cfg.ExcludeInterfaces) {
+		iptConfigurator.iptables.AppendRule(
+			constants.PREROUTING, constants.NAT, "-i", excludeInterface, "-j", constants.RETURN)
+		iptConfigurator.iptables.AppendRule(constants.OUTPUT, constants.NAT, "-i", excludeInterface, "-j", constants.RETURN)
```

Comment:
For TPROXY， maybe better apply it to mangle table

Reply:
Added.

Reply:
nit:  only set to nat when it is not TPROXY

Reply:
> nit: only set to nat when it is not TPROXY

@hzxuzhonghu Can you explain a little bit?

I add "shortcircuit" to NAT in TPROXY because I see TPROXY does use NAT table: See https://github.com/istio/istio/blob/master/tools/istio-iptables/pkg/cmd/testdata/tproxy.golden#L18-L19

The NAT shortcircuit should be added to NAT unless all existing NAT are not hit, correct?